### PR TITLE
Super_Admin Simpleform Compatibility Followup

### PR DESCRIPTION
**Pull request by AndrwM**
# Super_Admin Form Conversion Followup 🐍
#### Textarea
  - Shrink min-height
  - Error message support
  - simpleform error class support

#### Switches
  - Shrink margin between stacked switches

#### Select
  - Cleanup the preview implementation to require less variables
  - Remove the dependancy of using `[required]` and migrate to **a JS solution**
  - Refactor with a cleaner implementation and lean on JS to build some if the details
  - Better keyboard focus interaction

#### Sage Form
  - Refactor classes to new syntax.

#### Button
  - `align-end` helper

#### General
  - `isEmptyString` JS Helper

### CODE REVIEW

File: app/assets/stylesheets/sage/system/patterns/elements/_switch.scss [hunk 1]
```diff
@@ -11,6 +11,10 @@
   align-items: center;
   margin-bottom: $sage-form-element-spacing;
   color: $sage-switch-color-default-text;
+
+  & + & {
```
Comment (philschanely) [inline app/assets/stylesheets/sage/system/patterns/elements/_switch.scss:15]: 😂

**Review by philschanely — APPROVED** [review]

File: app/assets/stylesheets/sage/system/patterns/elements/_form_select.scss [hunk 2]
```diff
@@ -52,12 +31,29 @@ $-empty: "invalid";
 }
 
 .sage-select__label {
+  @extend %t-sage-body;
   position: absolute;
   white-space: nowrap;
   line-height: initial;
   pointer-events: none;
+  color: $-sage-selectfield-color-default;
+  line-height: inherit;
```
Comment (teenwolfblitzer) [inline app/assets/stylesheets/sage/system/patterns/elements/_form_select.scss]: Is this a duplicate of line 37?

**Review by teenwolfblitzer — APPROVED** [review]
👍 LGTM, one small item to check out

File: app/assets/stylesheets/sage/system/patterns/elements/_form_select.scss (hunk 2 above)
Comment (AndrwM) [reply app/assets/stylesheets/sage/system/patterns/elements/_form_select.scss]: Oh thank you, good catch 👍

**Pull request merged**